ganga/GangaCore/Core/GangaThread/WorkerThreads/WorkerThreadPool.py

```diff
@@ -12,6 +12,12 @@
 
 from collections import namedtuple
 
+try:
+    timeout = getConfig('Queues')['ShutDownTimeout']
```

Comment:
I do not see any reason for the `try ... except` code. When you have defined the new configuration object, you know that it will exist. So I siggest just to have the single line `timeout = getConfig('Queues')['ShutDownTimeout']`.